x-pack/legacy/plugins/maps/public/connected_components/map/feature_tooltip.js

```diff
@@ -18,16 +19,22 @@ import {
 import { i18n } from '@kbn/i18n';
 import { FeatureProperties } from './feature_properties';
 import { FormattedMessage } from '@kbn/i18n/react';
+import { GEO_JSON_TYPE, ES_GEO_FIELD_TYPE } from '../../../common/constants';
+import { FeatureGeometryFilterForm } from './feature_geometry_filter_form';
 
 const ALL_LAYERS = '_ALL_LAYERS_';
 const DEFAULT_PAGE_NUMBER = 0;
 
+const PROPERTIES_VIEW = 'propertiesView';
```

Comment:
Make this an enumeration and use CAPS (e.g. `const VIEWS = {PROPERTIES_VIEW: `PROPERTIES_VIEW`, GEOMETRY_FILTER_VIEW: 'GEOMETRY_FILTER_VIEW'}`) for consistency with other enumerations